### PR TITLE
71 - Bag: product-list-item-appearance

### PR DESCRIPTION
**Pull request by VVMaxVV**
Added shadow and margin between items in ProductsListFragment.kt
https://trello.com/c/c1BIiuUt/71-bugproduct-list-item-appearance

### CODE REVIEW

File: app/src/main/res/layout/item_product_category.xml [hunk 1]
```diff
@@ -1,77 +1,91 @@
...
-        android:textColor="@color/item_product_category_number_of_review_text_color"
-        android:textSize="@dimen/dimen_item_product_category_number_of_review_text_size"
-        android:text="@{product.numberOfReviews}"
-        app:layout_constraintBottom_toBottomOf="@+id/ratingBar"
-        app:layout_constraintStart_toEndOf="@+id/ratingBar"
-        app:layout_constraintTop_toTopOf="@+id/ratingBar"
-        tools:text="(10)" />
+        android:layout_width="match_parent"
+        app:cardCornerRadius="@dimen/dimen_item_category_card_corner_radius"
+        app:cardElevation="6dp">
```
Comment (darthoo) [inline app/src/main/res/layout/item_product_category.xml:17]: Use dimen for elevation too

File: app/src/main/res/layout/item_product_category.xml [hunk 2]
```diff
@@ -1,77 +1,91 @@
...
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/itemSimpleProdId"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:background="@color/white"
+            android:clickable="true"
```
Comment (darthoo) [inline app/src/main/res/layout/item_product_category.xml:24]: do we need this clickable attribute? As i rembemer onClick works even without this attr

File: app/src/main/res/layout/item_product_category.xml [hunk 3]
```diff
@@ -1,77 +1,91 @@
...
-        android:text="@{product.title}"
-        android:textSize="@dimen/dimen_item_product_category_title_text_size"
-        app:layout_constraintStart_toEndOf="@id/productImage"
-        app:layout_constraintTop_toTopOf="@id/productImage"
-        tools:text="Dorothy Perkins" />
+            <ImageView
+                android:id="@+id/productImage"
+                android:layout_width="@dimen/dimen_item_product_category_image_size"
+                android:layout_height="@dimen/dimen_item_product_category_image_size"
+                app:image_uri="@{product.imageUrl}"
```
Comment (darthoo) [inline app/src/main/res/layout/item_product_category.xml:31]: As I understand you have @BindingAdapter with attr `app:image_uri`. Change attr to `android:src`. Would be better to use attr to save image in image view.

**Review by darthoo — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/internetshop/presentation/activity/fragments/ProductsListFragment.kt [hunk 4]
```diff
@@ -57,7 +58,14 @@ class ProductsListFragment : BaseFragment() {
         val adapter = ProductsAdapter()
         val recyclerView = binding?.recyclerViewProducts
         recyclerView?.let {
-            it.addItemDecoration(VerticalSpaceItemDecoration(32))
+            it.addItemDecoration(
+                VerticalSpaceItemDecoration(
+                    context
```
[the remark this answers is not in the document]
Comment (VVMaxVV) [reply app/src/main/java/com/example/internetshop/presentation/activity/fragments/ProductsListFragment.kt:63]: fixed

File: app/src/main/res/layout/item_product_category.xml (hunk 3 above)
Comment (VVMaxVV) [reply app/src/main/res/layout/item_product_category.xml:31]: fixed

**Review by darthoo — APPROVED** [review]

**Pull request merged**